cf-custom-resources/lib/dns-cert-validator.js

```diff
@@ -478,13 +525,15 @@ const getAllAliases = function (aliases) {
   } catch (error) {
     throw new Error(`Cannot parse ${aliases} into JSON format.`);
   }
-  var aliasList = [];
-  for (var m in obj) {
-    aliasList.push(...obj[m]);
+  let aliasList = [];
+
+  for (const [_, aliases] of Object.entries(obj)) {
```

Comment:
just curious why are we making this change 🤔

Reply:
Not much tbh. Just b/c it has slightly more readability +  JSLint always complains with the message "The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype."